staging/src/k8s.io/apiserver/pkg/server/options/admission.go

```diff
@@ -58,13 +68,18 @@ func (a *AdmissionOptions) AddFlags(fs *pflag.FlagSet) {
 		"File with admission control configuration.")
 }
 
-// ApplyTo adds the admission chain to the server configuration
-// the method lazily initializes a generic plugin that is appended to the list of pluginInitializers
+// ApplyTo adds the admission chain to the server configuration.
+// In case admission plugin names were not provided by a custer-admin they will be prepared from the recommended/default values.
+// In addition the method lazily initializes a generic plugin that is appended to the list of pluginInitializers
 // note this method uses:
 //  genericconfig.LoopbackClientConfig
 //  genericconfig.SharedInformerFactory
 //  genericconfig.Authorizer
 func (a *AdmissionOptions) ApplyTo(c *server.Config, informers informers.SharedInformerFactory, pluginInitializers ...admission.PluginInitializer) error {
+	if len(a.PluginNames) == 0 {
+		a.PluginNames = a.enabledPluginNames()
```

Comment:
Can I get `pluginNames` as a local variable instead of rewriting the value as a side-effect of this method?